JobToResume.md

```diff
@@ -0,0 +1,69 @@
+Job To Resume
+==================
+
+#### Table of Contents
+_______
+
+- [Summary](#summary)
+- [Request structure](#request-structure)
+- [Response structure](#response-structure)
+- [Versioning](#versioning)
+
+## Summary
+
+Job To Resume Service recommends resumes based on job data. Job To Resume Service is available at 
+`/core/jobtoresume`.
+
+
+## Request Structure
+Requests consist of:
+
+|                   |               |
+|-------------------|---------------|
+|`carotene_id`      | string        | 
+|`carotene_version` | string        |
+|`skills`           | Array(Skill)  |
+|`location`         | Location      |
+
+Example cURL request:
+
+```
+curl -X POST \
+  https://api.careerbuilder.com/core/jobtoresume \
+  -H 'Accept: application/json;version=1.0' \
+  -H 'Authorization: <BEARER_TOKEN>' \
+  -H 'Content-Type: application/json' \
+  -d '{
+	"carotene_id": "11.0",
+	"skills": [{"skill_did": "KS120L16S3RLJ82VQ08H"}, {"skill_did": "KS120L96KMYTDJ48NRSH"}],
```

Comment:
Wait, skill objects are just `skill_did`s? Or can they have more fields? In that case, why an array of objects and not an array of strings?

Reply:
That was Lamar's decision.:)